org-formation/050-costs/_tasks.yaml

```diff
@@ -11,3 +11,14 @@ AnomalyDetectorService:
     Region: !Ref primaryRegion
   Parameters:
     Subscriber: !GetAtt CurrentAccount.RootEmail
+
+CostAndUsageReports:
+  Type: update-stacks
+  Template: cur.yaml
+  StackName: !Sub '${resourcePrefix}-CUR-Monthly'
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: true
+    Account: '*'
```

Comment:
this only needs to be deployed to the Master account so what you want is `Account: !Ref MasterAccount`

Reply:
Oh, we don't need to deploy it to all accounts in order to collect information on all accounts? Cool